Description: remove Red Hat specific menu categories
Author: Francois Marier <francois@debian.org>
Forwarded: not-needed
Last-Update: 2010-05-17

--- a/src/burgerspace.desktop.in
+++ b/src/burgerspace.desktop.in
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Name=@PACKAGE_FULL_NAME@
-Categories=X-Red-Hat-Base;Application;Game;ArcadeGame;
+Categories=Game;ArcadeGame;
 Comment=@PACKAGE_SUMMARY_EN@
 Comment[fr]=@PACKAGE_SUMMARY_FR@
 Exec=@PACKAGE@
